packages/app/src/main.ts

```diff
@@ -47,11 +47,56 @@ function createWindow() {
     }
   });
 
+  protocol.handle('vod', async (request) => {
+    const encodedFilename = decodeURI(request.url.slice('vod://wowarenalogs/'.length, request.url.length));
```

Comment:
what guarantees the host segment to be "wowarenalogs"? is it possible people would send vod://someotherstuff/ to this?

Reply:
The host segment is actually just a sentinel value to avoid Chrome's canonicalizing of the domain. Can go into more detail about this on discord but it's just to prevent the base64 string from being corrupted (via lowercasing) when the <video> element passes its request into Chrome.

The only real security feature we have on this is the .mp4 check -- I'm open to opinions on ways to further restrict the access of this schema to the file system but ultimately we need to be able to read video files in any folder the user has access to.